tabify/fecHeatmapTabify.js

```diff
@@ -35,16 +38,17 @@ export default class FecHeatmapTabify {
             const result = [];
             if (aggregations.date_histo && aggregations.date_histo.buckets) {
                 for (const dateHistoEntry of aggregations.date_histo.buckets) {
-                    const networkLossValue = dateHistoEntry.NetworkLoss && dateHistoEntry.NetworkLoss.value || 0.0;
-                    const lossAfterFecValue = dateHistoEntry.LossAfterFEC && dateHistoEntry.LossAfterFEC.value || 0.0;
+                    const networkLossValue = (dateHistoEntry.NetworkLoss && dateHistoEntry.NetworkLoss.value) || 0.0;
+                    const lossAfterFecValue = (dateHistoEntry.LossAfterFEC && dateHistoEntry.LossAfterFEC.value) || 0.0;
                     result.push({
                         key_as_string: dateHistoEntry.key_as_string,
                         date_histo: dateHistoEntry.key,
                         doc_count: 1,
                         stat: "Network Loss (%)",
                         key: networkLossValue,
-                        min: dateHistoEntry.MinNetworkLoss && dateHistoEntry.MinNetworkLoss.value || 0.0,
-                        max: dateHistoEntry.MaxNetworkLoss && dateHistoEntry.MaxNetworkLoss.value || 0.0,
+                        min: (dateHistoEntry.MinNetworkLoss && dateHistoEntry.MinNetworkLoss.value) || 0.0,
+                        max: (dateHistoEntry.MaxNetworkLoss && dateHistoEntry.MaxNetworkLoss.value) || 0.0,
+                        underlay: dateHistoEntry.UnderlayName.buckets && dateHistoEntry.UnderlayName.buckets.length && dateHistoEntry.UnderlayName.buckets[0].key || '',
```

Comment:
also include dateHistoEntry.UnderlayName check.
also add parenthesis similar to other props.
same in line 62

Reply:
Done.